packages/mdc-menu/simple/foundation.js

```diff
@@ -96,27 +116,23 @@ class MDCSimpleMenuFoundation extends MDCFoundation {
     /** @private {boolean} */
     this.isOpen_ = false;
     /** @private {number} */
-    this.startScaleX_ = 0;
+    this.openAnimationEndTimerId_ = 0;
```

Comment:
`openAnimationEndTimerId` is a number?

I think `openAnimationEndTimerId`, `closeAnimationEndTimerId`, *and* `selectedTriggerTimerId` all need better inline documentation for what they represent

Reply:
it is whatever, setTimeout returns and what you can pass to clearTimeout. Yeah it is a number - see https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout.